qutebrowser/misc/utilcmds.py

```diff
@@ -247,3 +247,33 @@ def log_capacity(capacity: int):
         raise cmdexc.CommandError("Can't set a negative log capacity!")
     else:
         log.ram_handler.change_log_capacity(capacity)
+
+
+@cmdutils.register(debug=True, name='debug-loglevel')
+def log_level(level=""):
```

Comment:
This should be called `debug_log_level` so the command is called `:debug-log-level` (all debug commands are prefixed with `:debug-*` currently).

Also, why is there a default argument to `level`? That would make the argument optional, which it isn't.